portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,46 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+html {
+  height: 100%;
+  width: 100%;
+}
+ 
+body {
+	background: linear-gradient(45deg,
+		rgba(20,18,19,.9) 30%,
+		rgba(0,94,105,.9) 100%),
+	url(Images/github_pattern.svg);
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	margin: 0;
+}
+ 
+#title {
+  text-align: center;
+}
+ 
+h1 {
+  color: white;
+}
+
+#text {
+	background: rgba(255,255,255,.7); 
+	display: inline-block;
+}
+
+#selfie-container {
+  width: 20em;
+}
+ 
+#selfie {
+	object-fit: contain;
+	max-width: 100%;
```

Comment:
nit: Sorry I missed this one too - switch these to be in alphabetical order :)

Reply:
Done!